### DISCUSSION TITLE
Copilot Chat not working on one desktop but works on others

### DISCUSSION THREAD

**Discussion by ceponatia** [community/community#57533] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hi, I've been using Copilot Chat for about a month. It's worked perfectly on my laptop and home Windows 11 desktop, but it doesn't work at all on my Windows 10 desktop at work. I'm trying to troubleshoot if this has to do with network settings or something else, but can't find any information online on where to find the error log.

Any time I send a prompt in Copilot Chat on my Windows 10 desktop, it returns "Error on conversation request. Check the log for more details." Where can I find this log?

Thank you

**Comment by mccormickk** [comment]
Press the Windows key on your keyboard or click on the Start button to open the Start menu.

In the search bar, type Event Viewer and press Enter.

The Event Viewer window will open. On the left-hand side, expand the Windows Logs folder.

You should see several log options, including Application, Security, Setup, System, and Forwarded Events. Click on Application.

Note: If you don't see the Application log, you can try checking the System log instead.

In the middle pane, you'll find a list of events. Look for the events with the source Copilot Chat or any related errors around the time you encountered the issue.

Double-click on the event to open its details. You can find more information about the error in the General tab.

**Reply by ceponatia** [reply]
Thanks, I found the Application log. I don't see anything related to Copilot or VSCode, though. I sent another prompt and got another error, but upon refreshing the log nothing new showed up. I also looked in the other Windows Logs folders to no avail. I don't have access to the Security logs due to organization restrictions. This may just be due to my office closing a port required by Copilot.

**Reply by mccormickk** [reply]
Try then to speak to the system administrator who manages the office

**Comment by starball5** [comment]
Potentially related: https://github.com/microsoft/vscode-copilot-release/issues/366

**Comment by ceponatia** [comment]
This seems to have been resolved.
